src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,87 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_STORAGE_LENGTH = 10;
+    private Pair<K, V>[] pairs = new Pair[MAX_STORAGE_LENGTH];
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```

Этот параметр будет проинициализирован нулём по умолчанию